f/connectors/arcgis/arcgis_feature_layer.py

```diff
@@ -0,0 +1,274 @@
+# requirements:
+# psycopg2-binary
+# requests~=2.32
+
+import json
+import logging
+import os
+from pathlib import Path
+
+import requests
+
+from f.common_logic.db_connection import postgresql
+from f.connectors.geojson.geojson_to_postgres import main as save_geojson_to_postgres
```

Comment:
You asked about calling another script from this script. I see no problem with it. Let's try it out and see how it goes.